### DISCUSSION TITLE
builtin_toolchains.md:38 错别字

### DISCUSSION THREAD

**Discussion by cosing** [xmake-io/xmake#4347] [General]
仅仅切花 c/c++ 编译器。
->
仅仅切换 c/c++ 编译器。

https://github.com/xmake-io/xmake-docs/blob/5ee301517cd46d33df9687d50a4c9cbc2ddfc4ce/zh-cn/toolchain/builtin_toolchains.md?plain=1#L38

**Comment by cosing** [comment]
发错地方了，这里怎么不是doc仓库

**Comment by cosing** [comment]
啊？为什么doc的问题和讨论会指向这里，
（并且doc仓库的issue模板很奇怪，还需要写版本什么的

**Reply by waruqi** [reply]
你直接点文档最上面的 edit document 编辑文档，就能提 pr 过来直接帮忙改了
